mirrord/kube/src/api/target_list.rs

```diff
@@ -0,0 +1,31 @@
+use k8s_openapi::api::core::v1::Pod;
+use kube::{api::ListParams, Api};
+
+use super::{get_k8s_api, kubernetes::create_kube_api};
+use crate::error::Result;
+
+pub async fn get_kube_pods(namespace: Option<&str>) -> Result<Vec<(String, Vec<String>)>> {
```

Comment:
I believe this belongs in the cli, as I don't see any other component using it?

Reply:
great idea!, ill change this